tsdb/index/tsi1/partition.go

```diff
@@ -97,7 +94,7 @@ func NewPartition(sfile *tsdb.SeriesFile, path string) *Partition {
 		seriesIDSet: tsdb.NewSeriesIDSet(),
 
 		// Default compaction thresholds.
-		MaxLogFileSize: DefaultMaxLogFileSize,
+		MaxLogFileSize: tsdb.DefaultMaxIndexLogFileSize,
```

Comment:
Shouldn’t this be `tsdb.MaxIndexLogFileSize`?

Reply:
`tsdb.MaxIndexLogFileSize` would indicate that there is a fixed setting. It's using the default value when initialized and then the `Index` overrides the value with its value here: https://github.com/influxdata/influxdb/blob/dc71a8d82bf94ce4ed62818760d6a28b1abf8996/tsdb/index/tsi1/index.go#L180

Reply:
Ah I see. Because I could see the `With` overrider in the chageset I missed that.

Reply:
Ah I see. Because I could see the `With` overrider in the chageset I missed that.